### PR TITLE
Add some missing supported PVs and a capability overview

### PR DESCRIPTION
**Pull request by tolleiv**
Hi,
the current documentation misses some supported PVs and I think showing the possible capabilities for all of them makes it easier to choose between storage solutions.

The PR adds missing PVs to the existing list and provides a PV<>capability table.

Cheers

### CODE REVIEW

File: docs/user-guide/persistent-volumes/index.md [hunk 1]
```diff
@@ -103,6 +108,23 @@ In the CLI, the access modes are abbreviated to:
 > __Important!__ A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time.
 
 
+|                      | ReadWriteOnce| ReadOnlyMany| ReadWriteMany|
```
Comment (johndmulhausen) [inline docs/user-guide/persistent-volumes/index.md]: Couldn't the blank column header here say: "Volume Type" or "Storage Driver" or "Provider"?

Comment (tolleiv) [inline docs/user-guide/persistent-volumes/index.md]: You're right - I used "Volume Plugin" to keep it consistent with the rest of the document.

**Comment by kelseyhightower** [comment]
@tolleiv Can we fix the ubuntu getting started guide in a separate commit? Once that is done, this will be ready for review.

**Comment by tolleiv** [comment]
@kelseyhightower thanks for the pointer, this wasn't even supposed to be part of any commit. I've adjusted the PR, so that just the PV listing is updated and all the rest is untouched

**Comment by kelseyhightower** [comment]
LGTM

**Pull request merged**